grpc/common/common.go

```diff
@@ -127,12 +127,20 @@ func (m *metric) Unit() string                  { return m.unit }
 
 // Convert common.Metric to core.Metric
 func ToCoreMetric(mt *Metric) core.Metric {
+	var lastAdvertisedTime time.Time
+	// if the lastAdvertisedTime is not set we handle.  -62135596800 represents the
+	// number of seconds from 0001-1970 and is the default value for time.Unix.
+	if mt.LastAdvertisedTime.Sec == int64(-62135596800) {
```

Comment:
Wow, good catch @nanliu . I haven't been aware of that some places in the world (maybe Poland too, gotta check) had in the past different and fractional "time zone" ...and PCs actually take this into account :100: